### DISCUSSION TITLE
VS code debugging

### DISCUSSION THREAD

**Discussion by dopplen-dev** [mindsdb/mindsdb#2642] [Get help]
I would to know if there is support for VS code style debugging . Currently it looks like , the HTTP flask API is spawned with a main loop and multiprocessing. Is it possible to start the services individually instead of the main loop.

**Comment by StpMax** [comment]
Hi @dopplen-dev ! What exactly issue you have with debugging in vscode? I also use it, and i can debug any process spawned in main.

**Reply by dopplen-dev** [reply]
thanks for the reply . It would seem that I was trying to use flask configuration for vs code debugging. but i understand now that you have to use module debugging.
i used this configuration 
{
    "version": "0.2.0",
    "configurations": [
    {"name":"Python: Module","type":"python","request":"launch","module":"mindsdb","justMyCode":true}
    ]
}

and is working now.
Please let me know if your config file is different / if there is a better configuration.
